pom.xml

```diff
@@ -75,6 +75,13 @@
         </configuration>
       </plugin>
       <plugin>
+      	<groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
```

Comment:
@siyuanh 
Is this a general comment or review comment?
Honestly, I did not get your feedback about what needs to be changed.